exercises/practice/perfect-numbers/.meta/config.json

```diff
@@ -32,8 +32,5 @@
   },
   "blurb": "Determine if a number is perfect, abundant, or deficient based on Nicomachus' (60 - 120 CE) classification scheme for positive integers.",
   "source": "Taken from Chapter 2 of Functional Thinking by Neal Ford.",
-  "source_url": "https://www.oreilly.com/library/view/functional-thinking/9781449365509/",
-  "custom": {
-    "ignore-count-ignores": true
-  }
```

Comment:
This is not necessary anymore since the generator is used instead of macros to create the test suite.